docs/layers/heatmap.md

```diff
@@ -0,0 +1,32 @@
+# HeatMaps
+
+This directory contains documentation on adding heatmap layers.\
+For full documentation follow this [link](https://docs.mapbox.com/help/tutorials/make-a-heatmap-with-mapbox-gl-js/).
+
+## Sample heatmap layer
+
+```
+{
+    "label": "Sample Layer",
+    "type": "heatmap",
+    "source": {...},
+    "priority": <number>,
+    "paint": {
+        "heatmap-weight": [...]
+        "heatmap-radius": [...],
+        "heatmap-opacity": [...],
+        "heatmap-color": [...]
+    }
```

Comment:
Could we also include what are the default values for these? I think we can probably just copy and paste directly from the Mabbox docs.